### DISCUSSION TITLE
有什么办法可以直接将效果应用在视频上吗？

### DISCUSSION THREAD

**Discussion by Nekoler** [Blinue/Magpie#943] [Q&A]
我想将各种效果应用在视频上，得到一个新的视频文件，便于在其他设备上查看，有什么方法可以实现吗？

我查到`FFmpeg`的`libplacebo`滤镜可以将glsl文件应用到视频上，但是尝试了几次，发现只会输出空文件

**Comment by Blinue** [comment]
超分视频不在 Magpie 的职能范围里。mpv 可能可以做到，你可以参考 https://github.com/hooke007/MPV_lazy/discussions/416
